#!/usr/bin/env python

PACKAGE = "rbot280"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add( name,  paramtype, level, description, default, min, max)
gen.add("desired_heading", double_t, 0, 'desired heading', 0.0, 0.0, 3.14)
gen.add("desired_speed", double_t, 0, 'desired velocity_x', 0.0, -5.0, 5.0)

exit(gen.generate(PACKAGE, "rbot280", "DesiredHdgSpdDynamic"))
